.github/workflows/action-send-ci-cd-status-slack-message.yml

```
@@ -0,0 +1,99 @@
+          RUN_URL: ${{ github.event.repository.html_url }}/actions/runs/${{ github.run_id }}
+          # statuses
+          INFRA_STATUS: "${{ steps.status-emojis.outputs.INFRA_EMOJI }}"
+          APPS_STATUS: "${{ steps.status-emojis.outputs.APPS_EMOJI }}"
+          SLACK_NOTIFIER_STATUS: "${{ steps.status-emojis.outputs.SLACK_NOTIFIER_EMOJI }}"
+          E2E_TESTS_STATUS: "${{ steps.status-emojis.outputs.E2E_TESTS_EMOJI }}"
+          SCHEMA_NPM_STATUS: "${{ steps.status-emojis.outputs.SCHEMA_NPM_EMOJI }}"
+          PUBLISH_STATUS: "${{ steps.status-emojis.outputs.PUBLISH_EMOJI }}"
+        uses: slackapi/slack-github-action@v1.27.0
+        with:
+          channel-id: ${{ secrets.SLACK_CHANNEL_ID_FOR_RELEASES }}
+          payload-file-path: "./.github/slack-templates/pipeline-failed.json"
```

yamllint

[error] no new line character at the end of file

(new-line-at-end-of-file)